codecov.yml

```diff
@@ -1,11 +1,14 @@
 codecov:
   notify:
-    after_n_builds: 10
```

Comment:
@thomasrockhu-codecov can I double check that removing this will not result in Codecov posting a comment?

Othwer than this, the change to informational status is 👍

Reply:
@zerolab yup that's correct, the yaml file still is setting

```
comment: off
```
further down below. This PR should mirror what you would expect to see going forward (the 2 passing status checks at the bottom)

Reply:
Thank you for the confirmation. LGTM